### PR TITLE
[Ready] Ports/Adds new berets but I don't break the branch somehow

### PR DESCRIPTION
**Pull request by Xantholne**
Ports some berets adds even more to most jobs

## About The Pull Request

Ports some berets from Paradise (I believe ported from elsewhere) as well as adding some new ones created by myself. New berets include
Cap, HOP, HOS, Warden,  CMO, Med, Chem, Viro, Robo, Sci, Atmo, Eng, Ce. 
Heads will find their new berets in their lockers, the rest will be found in those specific autodrobes for those jobs

## Why It's Good For The Game

This adds more player style choice for what exactly they wish to wear for outfits providing more variety since there does not seem to be a lot of head wear that is rather unique or something palyers would want to wear

## Changelog
:cl:
add: New Berets for most heads and departments available in their autodrobes or lockers
/:cl:

dreamseeker_tKyMo8IbRO

Top row left to right
HOS, Cap, HOP, Standard Purple, Standard Blue, Warden, CMO, Med, Chem, Viro, Sci, Robo
Second row
Atmos, Engi, Ce

No on mob picture since for some reason my uniforms are invisible and not really letting show how they match ¯\_(ツ)_/¯

### CODE REVIEW

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -63,7 +64,8 @@
 	product_ads = "Guaranteed to protect your feet from industrial accidents!;Afraid of radiation? Then wear yellow!"
 	vend_reply = "Thank you for using the EngiDrobe!"
 	products = list(/obj/item/clothing/accessory/pocketprotector = 5,
-					/obj/item/storage/backpack/duffelbag/engineering = 3,
+					/obj/item/clothing/head/beret/eng = 3,
+					/obj/item/storage/backpack/duffelbag/engineering = 2,
```
Comment (Linzolle) [inline code/modules/vending/wardrobes.dm]: Why is this lowered?

**Comment by Linzolle** [comment]
Lots of unnecessary variable overrides in here that are inherited from the parent type.

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -51,6 +51,14 @@
 
 	dog_fashion = null
 
+/obj/item/clothing/head/caphat/beret/white
+	name = "captain's white beret"
+	desc = "A white beret fit for a leader."
+	icon_state = "beret_captain_white"
+	dynamic_hair_suffix = ""
+
+	dog_fashion = null
```
Comment (Linzolle) [inline code/modules/clothing/head/jobs.dm]: dynamic_hair_suffix and dog_fashion don't need to be defined as they are inherited from /obj/item/clothing/head/caphat/beret

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -67,6 +75,14 @@
 
 	dog_fashion = null
 
+/obj/item/clothing/head/hopcap/beret/white
+	name = "head of personnel's white beret"
+	desc = "The symbol of true bureaucratic micromanagement, although in a fancy form."
+	icon_state = "beret_white_hop"
+	dynamic_hair_suffix = ""
+
+	dog_fashion = null
```
Comment (Linzolle) [inline code/modules/clothing/head/jobs.dm]: Same thing, desc also doesn't need to be overrided if it's the same.

File: code/modules/clothing/head/jobs.dm [hunk 4]
```diff
@@ -245,6 +276,13 @@
 	armor = list("melee" = 40, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
 	strip_delay = 60
 
+/obj/item/clothing/head/beret/sec/corporatewarden
+	name = "warden's corporate beret"
+	desc = "A special beret with the Warden's insignia emblazoned on it. For wardens with class."
+	icon_state = "beret_corporate_warden"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	strip_delay = 60
```
Comment (Linzolle) [inline code/modules/clothing/head/jobs.dm]: armor and strip_delay unnecessary here.

File: code/modules/vending/wardrobes.dm (hunk 1 above)
Comment (Xantholne) [reply code/modules/vending/wardrobes.dm]: Had moved to put beret at top so bags are still next to each other on the menu

Comment (Seris02) [reply code/modules/vending/wardrobes.dm]: I think they mean why did you lower the amount of bags

Comment (Xantholne) [reply code/modules/vending/wardrobes.dm]: Oh, accidental. Will have to fix.

**Review by silicons — APPROVED** [review]

**Pull request merged**